src/common/components/exchange/TxSummaryFixed.vue

```diff
@@ -352,6 +352,7 @@
               <p class="light-grayish">
                 {{ amount }} {{ currencyFromTicker }}
               </p>
+              <span v-if="type === txType.PEGOUT">USD {{ amountUSD }}</span>
```

Comment:
In pegin summary we use "$" instead of "USD". We should choose one or another and stick to it.

Reply:
We need to maintain USD to inform the correct coin for our user